python/ray/tests/test_failure.py

```diff
@@ -82,6 +82,38 @@ def f():
         assert False
 
 
+# Note that `ray_start_2_cpus` is required here. If we use
+# `ray_start_regular`, it's very likely that two tasks are submitted before
+# the first worker is registered to Raylet. Since
+# `maximum_startup_concurrency` is 1, worker pool will wait for the
+# registration of the first worker without starting any new workers. The
+# result is, the two tasks will be executed sequentially, which breaks an
+# assumption of this test case - the two tasks run in parallel.
+def test_get_throws_quickly_when_found_exception(ray_start_2_cpus):
+    @ray.remote
+    def bad_func1():
+        raise Exception("Test function intentionally failed.")
+
+    @ray.remote(max_retries=0)
+    def bad_func2():
+        eval("exit()")
+
+    @ray.remote
+    def slow_func():
+        time.sleep(10)
+
+    bad_functions = [bad_func1, bad_func2]
+    for bad_func in bad_functions:
```

Comment:
```suggestion
    for bad_func in [bad_func1, bad_func2]:
```
and remove above line